radis/lbl/factory.py

```diff
@@ -507,6 +523,16 @@ def __init__(
             if self.verbose >= 3:
                 print("DLM used. Defaulting broadening method to FFT")
             # TODO: make it a proper parameter in self.misc or self.params
+            
+            if awL_kind is None:
+                self.misc.awL_kind = 2
+            else:
+                self.misc.awL_kind = awL_kind
```

Comment:
Should try with : 

```
self.misc['awL_kind'] = awL_kind 
```

This should fix the [current KeyError](https://travis-ci.com/github/radis/radis/jobs/365053344#L1471) in the tests.